src/icons/file/README.md

```diff
@@ -21,111 +21,107 @@ const IconsExample = require('../../../examples/src/IconsExample').default;
 const icons = [
     {
         name: 'IconFileAudio',
-        component: require('./IconFileAudio').default,
+        component: require('../imported/Icon/IconFileAudio32').default,
```

Comment:
We don't currently differentiate imported icons, nor do we use PascalCase for directory names. It seems like these new components should live directly in `src/icons/file` rather than `src/icons/imported/Icon`.

Reply:
There will eventually be other assets being imported that aren't specifically file or folder icons, so I wanted to sequester the new ones from all the others for that and a few other reasons:

- since the organization of the source asset repository might change, the import clears the contents then rebuilds it
- the directory structure (including the naming) is copied from the source repository as-is. File names are "PascalCase"d for consistency with component naming.

If we object specifically to `Icon`, I could request that change, and the imports would become `../imported/another-path/`... to whatever depth was suggested.

Reply:
@jstoffan perhaps using something other than "imported" is better here? Other ideas (considering the details I mentioned above)

Reply:
So the tool we're using to import images clears the output directory each time, basically? That's not ideal, but if it's unavoidable, we should still try to respect the organization of the icon directories. Can we use `src/icons/file/imported` for now?